ch_tools/chadmin/internal/table.py

```diff
@@ -1,6 +1,17 @@
-from click import ClickException
+import os
+import uuid
 
+from click import ClickException, Context
+
+from ch_tools.chadmin.cli.data_store_group import (
```

Comment:
Lets move these functions and constants into separate module. `ch_tools/chadmin/internal/clickhouse-disks.py` for example

Reply:
Ok. (filename clickhouse_disks.py)